spec/formatting.md

```diff
@@ -123,21 +127,26 @@ Finally, sort the list of variants `vars` and select the _pattern_:
       1. Let `matchpref` be an integer with the value `minpref`.
       1. Let `key` be the `tuple` _variant_ key at position `i`.
       1. If `key` is not the catch-all key `'*'`:
-         1. Let `ks` be the decoded value of `key`.
+         1. Assert that `key` is a _literal_.
+         1. Let `ks` be the resolved value of `key`.
          1. Let `matchpref` be the integer position of `ks` in `matches`.
       1. Set the `tuple` integer value as `matchpref`.
-   1. Call the method SortVariants(`sortable`).
+   1. Set `sortable` to be `SortVariants(sortable)`.
```

Comment:
Nitpicky, but let's be a bit more explicit here, and follow the same style of expression as on line 83.

```suggestion
   1. Set `sortable` to be the result of calling the method `SortVariants(sortable)`.
```